exercises/occams-razor.md

```diff
@@ -74,118 +81,116 @@ var learnConcept = function(examples, testQuery) {
  }); 
 }
 
-var examples = [3]
-var testQuery = 12
-var posterior = learnConcept(examples, testQuery)
-marginalize(posterior, function(x) {return x.hypothesis})
+var examples = [3, 9];
+var testQuery = 12;
+var posterior = learnConcept(examples, testQuery);
+viz.marginals(posterior);
+~~~~
+
+### Exercise 1.2
+
+Modify the model to include hypotheses that the numbers were generated by sampling from an interval.
```

Comment:
It would be good to still include the phrase "similarity-based" somewhere in this part, since that's the key cognitive idea...